### PR TITLE
21857/add user cabinet

### PR DESCRIPTION
**Pull request by StanislavKharkiv**

### CODE REVIEW

File: webviewer/src/pages/Account/Account.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { ContactUs } from 'components/ContactUs';
+import { Sidebar } from './components';
+import { Wrapper } from './Account.styles';
+import { PageContent } from './components';
+const TABS = [
+  { name: 'Profile', icon: 'Settings' },
+  { name: 'Transactions', icon: 'Document' }
+];
+
+export const Account = ({ ...props }) => {
```
Comment (avakin) [inline webviewer/src/pages/Account/Account.js:12]: image
is such padding value was into mockups? looks not good, maybe will be better to make them equal?

File: webviewer/src/pages/Account/components/PageContent/PageContent.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Profile } from '../Profile';
+import { PageWrapper } from '../PageWrapper';
+
+export const PageContent = ({ activeTab, tabs }) => {
+  switch (activeTab) {
+    case tabs[0].name:
```
Comment (avakin) [inline webviewer/src/pages/Account/components/PageContent/PageContent.js:7]: using indexes in such cases is not the best practice. 
maybe to use object with tabKey: tabConfig or something else to use some local ID for the tab. 
and then you will need to use tabs array, you can use Object(TABS).values() and receive the array that you want to map

File: webviewer/src/pages/Account/components/PageWrapper/PageWrapper.styles.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { em } from 'styles';
+import { Typography } from 'components/_shared/Typography';
+
+export const Header = styled(Typography).attrs({
+  element: 'h2',
+  variant: 'h2'
```
Comment (avakin) [inline webviewer/src/pages/Account/components/PageWrapper/PageWrapper.styles.js:7]: you don't need to use h$n, when you don't have  h$n-1 beforehead
h2 without h1 is not applicable

File: webviewer/src/pages/Account/components/Sidebar/Sidebar.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+export const Sidebar = props => {
+  const { activeTab, setActiveTab, tabs } = props;
+  const history = useHistory();
+  const handleClickHome = () => history.push(ROUTES.ROOT);
+
+  return (
+    <StyledSidebar>
+      <BreadCrumbs>
+        <BreadCrumbsItem onClick={handleClickHome}>Home</BreadCrumbsItem> /{' '}
+        <BreadCrumbsItem>Personal account</BreadCrumbsItem>
```
Comment (avakin) [inline webviewer/src/pages/Account/components/Sidebar/Sidebar.js:16]: please remove `cursor:pointer` from the last/active breadcrumb

File: webviewer/src/pages/Main/PageMain.jsx [hunk 5]
```diff
@@ -6,22 +6,21 @@ import { AreasSidebar } from 'components/Areas';
 import { Map } from 'components/Map';
 import { ContactUs } from 'components/ContactUs';
 
-import { useAreasActions, useUserActions } from 'state';
+import { useAreasActions } from 'state';
 
 export const PageMain = ({ ...props }) => {
-  const { getCurrentUser } = useUserActions();
   const { getAreas } = useAreasActions();
+  const isOpen = props.history.action === 'PUSH' && props.location.state?.isOpenSidebar;
```
Comment (avakin) [inline webviewer/src/pages/Main/PageMain.jsx:13]: useMemo?

**Review by avakin — CHANGES_REQUESTED** [review]
check posted comments, please

File: webviewer/src/pages/Main/PageMain.jsx (hunk 5 above)
Comment (StanislavKharkiv) [reply webviewer/src/pages/Main/PageMain.jsx:13]: I don't see the point. It's will not improve performance, but will only add an extra wrapper function

File: webviewer/src/pages/Account/components/PageWrapper/PageWrapper.styles.js (hunk 3 above)
Comment (StanislavKharkiv) [reply webviewer/src/pages/Account/components/PageWrapper/PageWrapper.styles.js:7]: ok, changed to h1

File: webviewer/src/pages/Account/Account.js (hunk 1 above)
Comment (StanislavKharkiv) [reply webviewer/src/pages/Account/Account.js:12]: yes it's like in design, padding is different

File: webviewer/src/pages/Account/components/PageContent/PageContent.js (hunk 2 above)
Comment (StanislavKharkiv) [reply webviewer/src/pages/Account/components/PageContent/PageContent.js:7]: fixed

File: webviewer/src/pages/Account/components/Sidebar/Sidebar.js (hunk 4 above)
Comment (StanislavKharkiv) [reply webviewer/src/pages/Account/components/Sidebar/Sidebar.js:16]: fixed

**Review by avakin — APPROVED** [review]
LGTM

**Review by adolgarev — APPROVED** [review]

**Pull request merged**